docs/src/content/docs/guides/i18n.mdx

```diff
@@ -141,11 +141,13 @@ If a translation is not yet available for a language, Starlight will show reader
 
 ## Translate Starlight's UI
 
-import I18nList from '../../../components/i18n-list.astro';
+import LanguagesList from '../../../components/languages-list.astro';
 
 In addition to hosting translated content files, Starlight allows you to translate the default UI strings (e.g. the "On this page" heading in the table of contents) so that your readers can experience your site entirely in the selected language.
 
-<I18nList /> translated UI strings are provided out of the box, and we welcome [contributions to add more default languages](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
+<LanguagesList /> translated UI strings are provided out of the box, and we
```

Comment:
Nice — thanks for adding the prop!

While not strictly speaking required in English because the languages get capitalized anyway, what do you think about including the prop here for correctness and especially because this will be the version people base their own translation updates on?

```suggestion
<LanguagesList startsSentence /> translated UI strings are provided out of the box, and we
```

Reply:
Ohhh, that actually makes a lot of sense, thanks for the great suggestion!